alerts/resource_alerts.libsonnet

```diff
@@ -82,7 +82,7 @@
             expr: |||
               sum(kube_resourcequota{%(prefixedNamespaceSelector)s%(kubeStateMetricsSelector)s, type="hard", resource="memory"})
                 /
-              sum(kube_node_status_allocatable_memory_bytes{%(nodeExporterSelector)s})
+              sum(kube_node_status_allocatable_memory_bytes{%(prefixedNamespaceSelector)s%(kubeStateMetricsSelector)s})
```

Comment:
`%(prefixedNamespaceSelector)s` only applies to namespaced APIs, which the node API is not, so we don't need that part here.

Reply:
Fixed, thanks.